src/app/core/signals/remote_signals/connector.nim

```diff
@@ -21,6 +21,8 @@ type ConnectorGrantDAppPermissionSignal* = ref object of Signal
   url*: string
   name*: string
   iconUrl*: string
```

Comment:
These arguments were missing from the integration. We need to know in the UI on what account a dapp is connected and on what chains.